pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "cps"
+version = "0.dev"
+description = ""
+authors = ["The CPS Project <@>"]
+readme = "README.md"
+packages = [
+  { include = "cps.py", from = "_extensions" },
+  { include = "autosectionlabel.py", from = "_extensions" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+
+[tool.poetry.group.docs.dependencies]
+python = "^3.10"
+sphinx = ">=7.2.6"
```

Comment:
This suggests we should also update the version requirement in `conf.py`. (Also... 7.2.6? I only have 6.2.1! And I feel like we should maybe pin this to `<8`, unless `>=` does that already?)

Reply:
Whee, seems they've been busy making releases! 6.2.1 is less than a year old.

Okay, so maybe we should use `>= 6.2, < 7.3`? Or do we want to allow all the way back to 5.3, as per `conf.py`? If we're effectively raising the minimum here, we should probably change `conf.py` to match.

(Speaking of `conf.py`, I think that `sys.path` tweak still needs updating? :slightly_smiling_face:)

Reply:
Most of the changes between then and now don't affect us, but we can do that range sure 🙂